project-words.txt

```diff
@@ -1,63 +1,216 @@
-Vfile
-Zenodo
-GOTBIN
-Omics
-NBIS
-Mihail
-Hansson
-metabolicatlas
-kegg
-gotenzymes
-Recon
-subsys
-vuex
-pharmacogenomic
-Omix
-ENSG
-Cytoscape
+Agren
+Albornoz
+Altmetric
+apoc
+arrowless
+Avlant
+Banaei
+Benfeitas
+bigg
+Billa
+Bioconductor
+bioschemas
+boxe
+bulma
+Bulma
+ccms
+centering
+cerevisiae
+chebi
+CHEBI
+Cholley
+cmid
+cmids
 cmodel
-PNAS
-glyoxalate
+coelicolor
 comparemodels
-SULT
-ENSG
+compm
+compo
+connectdevelop
+csvg
+csvgs
+cutrectangle
+Cvijovic
+cxttap
+cypan
+Cytoscape
+cyzoom
+datasource
+Dina
+Domenzain
+donut
+Dviewer
+eaaz
+eccode
+elems
+Elems
+ensembl
+Ensembl
 ENSG
-PARP
+Esfahani
+europepmc
+Feizi
+fluxomics
+forcelayout
+Fruitfly
+fullhd
+fullheight
+fulltext
+gapless
+Gatto
+geneset
+genesets
```

Comment:
We have both "gene set" and "gene set", should we go for one of them?
`Resources.vue`: "The Kiwi module combines _geneset_ analyses with biological networks to visualize the interactions between _genesets_" and also "Platform for integrative analysis of omics data (PIANO) is an An R/Bioconductor package for _gene set_ analysis',"

Reply:
To my knowledge, this should be spelled `gene set`, with a space.

Reply:
Fixed in c24879bf40299a8156fddde78576b21ba6825c5a, based on the description in [their repo](https://github.com/SysBioChalmers/Kiwi)

Reply:
I added 136a44a, so that we consistently use the same spelling on the page. Feel free to revert if you don't approve.